### ISSUE TITLE
Passive losses from schedule E

### ISSUE THREAD

**Issue by zakpatterson** [ustaxes/UsTaxes#294]
If schedule E creates a net loss, then a negative number is brought forward to the 1040 which reduces taxable income. 

If schedule E has a net passive loss, then form 8582 must be used to figure the negative number that may be brought forward to the 1040.

**Comment by thegrims** [comment]
was this addressed by https://github.com/ustaxes/UsTaxes/pull/283 ?

**Comment by zakpatterson** [comment]
> was this addressed by https://github.com/ustaxes/UsTaxes/pull/283 ?

No, #283 only added a calculation to figure what amount of the net income from Schedule E contributes to the investment income cap for qualifying for EIC. This issue deals with the passive activity net loss limitation. Basically passive investment activities that create a net loss are only able to subtract from other passive activities. 

Real estate commissions for example are not deductible from your W2 income. Need to add form 8582 as an attachment.
